src/App.js

```diff
@@ -11,33 +11,132 @@ class App extends React.Component {
       cardDeck: makeShuffledDeck(),
       // currCards holds the cards from the current round
       currCards: [],
+      // game start
+      gameStart: false,
+      // round winner
+      roundWinner: null,
+      // player 1 score
+      player1Score: 0,
+      // player 2 score
+      player2Score: 0,
+      // game round number
+      gameRoundNumber: 0,
+      // plyaer 1 score
+      player1WinGameScore: 0,
+      // player 2 score
+      player2WinGameScore: 0,
     };
   }
 
+  resetGame = () => {
+    this.setState((prevState) => ({
+      cardDeck: makeShuffledDeck(),
+      currCards: [],
+      gameStart: false,
+      roundWinner: null,
+      player1Score: 0,
+      player2Score: 0,
+      gameRoundNumber: prevState.gameRoundNumber + 1,
+      player1WinGameScore:
+        prevState.player1Score > prevState.player2Score
+          ? prevState.player1WinGameScore + 1
+          : prevState.player1WinGameScore,
+      player2WinGameScore:
+        prevState.player2Score > prevState.player1Score
+          ? prevState.player2WinGameScore + 1
+          : prevState.player2WinGameScore,
+    }));
+  };
+
   dealCards = () => {
+    const newCurrCards = this.state.cardDeck.slice(-2);
+    let newRoundWinner = null;
+    if (newCurrCards[0].rank > newCurrCards[1].rank) {
+      newRoundWinner = 1;
+    }
+    if (newCurrCards[0].rank < newCurrCards[1].rank) {
+      newRoundWinner = 2;
+    }
+
     this.setState((state) => ({
       // Remove last 2 cards from cardDeck
       cardDeck: state.cardDeck.slice(0, -2),
-      // Deal last 2 cards to currCards
-      currCards: state.cardDeck.slice(-2),
+      currCards: newCurrCards,
+      roundWinner: newRoundWinner,
+      gameStart: true,
+      player1Score:
+        newRoundWinner === 1 ? state.player1Score + 1 : state.player1Score,
+      player2Score:
+        newRoundWinner === 2 ? state.player2Score + 1 : state.player2Score,
     }));
   };
 
   render() {
-    const currCardElems = this.state.currCards.map(({ name, suit }) => (
+    // Dealt Cards
+    const currCardElems = this.state.currCards.map(({ name, suit }, index) => (
       // Give each list element a unique key
-      <div key={`${name}${suit}`}>
-        {name} of {suit}
+      <div className="play-table" key={`${name}${suit}`}>
+        <p>Player {index + 1}'s cards:</p>
+        <p>
+          {name} of {suit}
+        </p>
+        <img
+          className="card-image"
+          src={require(`../src/images/${name}_of_${suit}.png`)}
+          alt="cardimage"
+        />
       </div>
     ));
 
+    // Game Win Count Tracking Messages
+    const roundWinnerMessage = this.state.roundWinner
+      ? `Player ${this.state.roundWinner} won this round!`
+      : `This round is a tie! Continue the game!`;
+    const player1WinCountMessage = `Player 1 has won ${this.state.player1Score} rounds in this game!`;
```

Comment:
If you feel all these strings are a chore to define, you can also do access state inline within your HTML.